core/ulog/README.md

```diff
@@ -1,8 +1,8 @@
 # Logging package
 
-ulog provides an API wrapper around the logging library (zap Logger)
-The logger is instantiated as logger with default options and can be configured
-via `Configure()` API and provided YAML configuration.
+`package ulog` provides an API wrapper around the logging library (zap Logger).
```

Comment:
- Remove "Logger" and maybe add a link to [zap](https://github.com/uber-go/zap) OR make it zap.Logger (I can't tell which one is more relevant from zap's docs)
- Add some high-level benefit for users up here (why should anyone read about an API wrapper around zap?)